### PR TITLE
report any (unless explicitly ignored) 4xx or 5xx error

### PR DESCRIPTION
**Pull request by jmonster**
previously only reported on `isBoom`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,10 +8,6 @@ exports.register = function (server, options, next) {
   server.plugins.icecreambar =  rollbar;
   server.plugins.icecreambar['default'] = rollbar;
 
-  server.on('request-error', function internalError (request, error) {
```
Comment (tevanoff) [inline index.js:11]: Why was this removed?

Comment (jmonster) [reply index.js:11]: previously `onPreRequest` didn't report `500` errors and `request-error` did; but turns out `request-error` isn't called if you have any `onPreRequest` hooks that change the reply to a non-error -- e.g. like we are doing in backboard to suppress users from seeing analytics failures.

So, I updated the `onPreRequest` in this project to report 500s now.
